doc/scapy/troubleshooting.rst

```diff
@@ -62,8 +62,8 @@ Getting help
 
 Common problems are answered in the FAQ.
 
-There's a low traffic mailing list at ``scapy.ml(at)secdev.org``  (`archive <http://news.gmane.org/gmane.comp.security.scapy.general>`_, `RSS, NNTP <http://gmane.org/info.php?group=gmane.comp.security.scapy.general>`_). You are encouraged to send questions, bug reports, suggestions, ideas, cool usages of Scapy, etc. to this list. Subscribe by sending a mail to ``scapy.ml-subscribe(at)secdev.org``.
```

Comment:
I thought this maillist was inactive, but from what I've checked it is still working
![image](https://user-images.githubusercontent.com/10530980/37111004-dd0da9b2-223e-11e8-9483-c956b6d82008.png)

Should we leave the link to it ?

Reply:
I don't use it but I don't think that we can remove it. 

Also, can you add a reference to the gitter channel ?

Reply:
Sure